src/main/java/com/nashtech/exception/ResourceNotFoundException.java

```diff
@@ -0,0 +1,20 @@
+package com.nashtech.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+/**
+ The ResourceNotFoundException class is a custom exception
+ that is thrown when a requested resource is not found.
+ */
+@ResponseStatus(value = HttpStatus.NOT_FOUND, reason = "Record not found")
```

Comment:
If in the exception handler you are hardcoding the status code then what is the relevance of status code here?